src/commands/game_options/timer.ts

```diff
@@ -24,8 +26,11 @@ const logger = new IPCLogger("guessTimeout");
 
 // eslint-disable-next-line import/no-unused-modules
 export default class GuessTimeoutCommand implements BaseCommand {
-    static TIMER_MIN_VALUE = 2;
+    static TIMER_MIN_VALUE = 0.75;
+    static TIMER_DEFAULT_VALUE = 2;
+    static TIMER_MIN_VALUE_NON_CLIP = 2;
     static TIMER_MAX_VALUE = 180;
+    static TIMER_MAX_ACCEPTABLE_CLIP_BEFORE_RESET = 5;
```

Comment:
wat is this

Reply:
Since the timer is being reused, it should fall back to the default clip value when beyond this. Otherwise, clip mode would play very long clips of a song, making the feature redundant